src/__tests__/parserAttributes.test.ts

```diff
@@ -0,0 +1,35 @@
+import { parse } from '../parser';
+
+test('attributes', () => {
+  expect(parse('abc<a href="/url/path" target="_blank" class="hello motos"><b>tagtext</b></a>')).toEqual([
```

Comment:
there should be a test with self-closing tag, full url like `href="https://abc.de/ab/cd?p1=a&p2=b4#top"` and potentially params without content `<input autofocus/>`